adapters/adot/adottest/supplemental/simple-audio.json

```diff
@@ -0,0 +1,69 @@
+{
+  "mockBidRequest": {
+    "id": "unsupported-audio-request",
+    "imp": [
+      {
+        "id": "unsupported-audio-imp",
+        "audio": {
+          "mimes": ["video/mp4"]
+        },
+        "ext": {
+          "bidder": {
+            "placementId": 1
+          }
+        }
+      }
+    ]
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "https://dsp.adotmob.com/headerbidding/bidrequest",
+        "body": {
+          "id": "unsupported-audio-request",
+          "imp": [
+            {
+              "id": "unsupported-audio-imp",
+              "audio": {
+                "mimes": ["video/mp4"]
+              },
+              "ext": {
+                "bidder": {
+                  "placementId": 1
+                }
+              }
+            }
+          ]
+        }
+      },
+      "mockResponse": {
+        "status": 200,
+        "body": {
+          "status": 200,
+          "body": {
```

Comment:
It looks like you've got `status` and `body` twice. Can you fix so there's only one occurrence of each?